platform/platform-commands/src/test/java/org/codice/ddf/commands/platform/ConfigStatusCommandTest.java

```diff
@@ -0,0 +1,157 @@
+                CONFIG_DIR.toString()) {
+            @Override
+            PrintStream getConsole() {
+                return mockConsole;
+            }
+        };
+
+        // Perform Test
+        configStatusCommand.doExecute();
+
+        // Verify
+        ArgumentCaptor<String> argument = ArgumentCaptor.forClass(String.class);
+        verify(mockConsole, times(4)).print(argument.capture());
+        List<String> values = argument.getAllValues();
+        assertThat(values.get(0), is(ERROR_RED));
+        assertThat(values.get(1), is(String.format(ConfigStatusCommand.FAILED_IMPORT_MESSAGE,
+                CONFIG_DIR.resolve(FAILED_CONFIG_FILE1.getFileName()).toString(),
+                FAILED_CONFIG_FILE1.toString())));
+        assertThat(values.get(0), is(ERROR_RED));
+        assertThat(values.get(3), is(String.format(ConfigStatusCommand.FAILED_IMPORT_MESSAGE,
+                CONFIG_DIR.resolve(FAILED_CONFIG_FILE2.getFileName()).toString(),
+                FAILED_CONFIG_FILE2.toString())));
+    }
+
+    @Test
+    public void testDoExecuteNoFailedImports() throws Exception {
+        // Setup
+        when(mockConfigStatusService.getImportStatus()).thenReturn(new ArrayList<>(0));
+        ConfigStatusCommand configStatusCommand = new ConfigStatusCommand(mockConfigStatusService,
+                CONFIG_DIR.toString()) {
+            @Override
+            PrintStream getConsole() {
+                return mockConsole;
+            }
+        };
+
+        // Perform Test
+        configStatusCommand.doExecute();
+
+        // Verify
+        ArgumentCaptor<String> argument = ArgumentCaptor.forClass(String.class);
+        verify(mockConsole, times(2)).print(argument.capture());
+        List<String> values = argument.getAllValues();
+        assertThat(values.get(0), is(SUCCESS_GREEN));
+        assertThat(values.get(1),
+                is(String.format(ConfigStatusCommand.SUCCESSFUL_IMPORT_MESSAGE, CONFIG_DIR)));
+    }
+
+    @Test
+    public void testDoExecuteExceptionWhileReadingFailedDirectory() throws Exception {
+        // Setup
+        doThrow(new IOException()).when(mockConfigStatusService).getImportStatus();
+        ConfigStatusCommand configStatusCommand = new ConfigStatusCommand(mockConfigStatusService,
+                CONFIG_DIR.toString()) {
+            @Override
+            PrintStream getConsole() {
+                return mockConsole;
+            }
+        };
+
+        // Perform Test
+        configStatusCommand.doExecute();
+
+        // Verify
+        ArgumentCaptor<String> argument = ArgumentCaptor.forClass(String.class);
+        verify(mockConsole, times(2)).print(argument.capture());
+        List<String> values = argument.getAllValues();
+        assertThat(values.get(0), is(ERROR_RED));
+        assertThat(values.get(1),
+                containsString("An error was encountered while executing this command."));
+    }
+}
```

Comment:
May want to add tests for the constructor arguments, i.e., not null, invalid directory (which may not apply if we take it out), etc.